qiling/core.py

```diff
@@ -547,6 +551,8 @@ def verbose(self):
     @verbose.setter
     def verbose(self, v):
         self._verbose = v
+        if self._verbose == -1:
+            self.log.disabled = True
```

Comment:
`Logger.disabled` is undocumented attributed. Use `Logger.setLevel` instead. Besides, the if statement shall go to `ql_resolve_logger_level`.